src/main/java/com/mercadopago/client/MercadoPagoClient.java

```diff
@@ -33,32 +39,64 @@ public MercadoPagoClient(MPHttpClient httpClient) {
     defaultHeaders.put(Headers.PRODUCT_ID, MercadoPagoConfig.PRODUCT_ID);
     defaultHeaders.put(
         Headers.USER_AGENT,
-        String.format("MercadoPago Java SDK %s", MercadoPagoConfig.CURRENT_VERSION));
-    defaultHeaders.put(Headers.CONTENT_TYPE, ACCEPT_HEADER_VALUE);
+        String.format("MercadoPago Java SDK/%s", MercadoPagoConfig.CURRENT_VERSION));
+    defaultHeaders.put(Headers.TRACKING_ID, MercadoPagoConfig.TRACKING_ID);
+    defaultHeaders.put(Headers.CONTENT_TYPE, CONTENT_TYPE_HEADER_VALUE);
   }
 
   protected MPResponse send(MPRequest request) throws MPException {
-    for (Map.Entry<String, String> entry : defaultHeaders.entrySet()) {
-      request.addHeader(entry.getKey(), entry.getValue());
-    }
-    if (!request.getUri().contains("/oauth/token")) {
-      request.addHeader("Authorization", String.format("Bearer %s", getAccessToken(request)));
-    }
-    return httpClient.send(addIdempotencyKey(request));
+    addDefaultHeaders(request);
+    addDefaultTimeouts(request);
+    addQueryParams(request, null);
+    addIdempotencyKey(request);
+
+    return httpClient.send(request);
   }
 
-  protected MPResponse send(String path, HttpMethod method, JsonObject payload) throws MPException {
-    MPRequest mpRequest = new MPRequest(UrlFormatter.format(path), method, defaultHeaders, payload);
-    return this.send(mpRequest);
+
+  protected MPResponse send(String path, HttpMethod method, Map<String, Object> queryParams, JsonObject payload) throws MPException {
+    return this.send(path, method, payload, queryParams, null);
   }
 
   protected MPResponse send(
-      String path, HttpMethod method, JsonObject payload, Map<String, String> headers)
+      String path, HttpMethod method, JsonObject payload, Map<String, Object> queryParams, MPRequestOptions requestOptions)
```

Comment:
Já que Java não tem parâmetros nomeados, seria interessante ajustar a assinatura desse método para seguir a ordem do método de cima
```suggestion
      String path, HttpMethod method, Map<String, Object> queryParams, JsonObject payload, MPRequestOptions requestOptions)
```

Reply:
Tinha iniciado a review antes de você ajustar 😅
Já tá certo agora